lib/employees/resolvers.ts

```diff
@@ -119,3 +123,35 @@ export const updateEmployee: Resolver = async (_, args, { prisma }) => {
     ok: true,
   };
 };
+
+/**
+ * Delete an employee
+ * @returns status of operation (ok)
+ */
+export const deleteEmployee: Resolver = async (_, args, { prisma }) => {
+  const { input } = args;
+  const { id } = input;
+  const employee = await prisma.employee.findUnique({
+    where: {
+      id: parseInt(id),
+    },
+    select: {
+      active: true,
+    },
+  });
+
+  if (employee == null) {
+    throw new EmployeeNotFoundError(`Employee with ID ${parseInt(id)} not found`);
+  } else if (!employee.active) {
+    throw new EmployeeAlreadyDeletedError(`Employee with ID ${parseInt(id)} already deleted`);
+  } else {
+    const employee = await prisma.employee.update({
+      where: { id: parseInt(id) },
+      data: { active: false },
+    });
+
+    if (employee.active) throw new ApolloError('Employee was unable to be deleted');
```

Comment:
Pretty sure that `employee` is not in sync with the database so I don't think this will work?

It would be better to take everything in the `else` block out of it and put it in a `try-catch` block instead with the `ApolloError` being thrown in the `catch` block.